builder/tests/common-types/src/main/java/io/helidon/common/types/TypedElementInfoSupport.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2023 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.common.types;
+
+import java.util.stream.Collectors;
+
+import io.helidon.builder.api.Prototype;
+
+final class TypedElementInfoSupport {
+    private TypedElementInfoSupport() {
+    }
+
+    @Prototype.PrototypeMethod
+    @Prototype.Annotated("java.lang.Override")
+    static String toString(TypedElementInfo me) {
```

Comment:
Will these prototype methods eventually be returning `ClassModel`? Seems to make the most sense..

Reply:
no, this is not generating the code, it is actually the code used at runtime referenced from the generated code.